classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+  end
```

Comment:
- [ ] After adding the `student` to the students array we also need to assign this classroom to the student. Kindly do that.

  ## SUGGESTION
  ```
  student.classroom = self
  ```